pallets/stream/src/lib.rs

```diff
@@ -490,6 +573,8 @@ pub mod pallet {
 				ensure!(stream_details.registry == registry_id, Error::<T>::UnauthorizedOperation);
 			}
 
+			<UniqueProofs<T>>::remove(&stream_id, stream_details.digest);
```

Comment:
This needs to be a loop. An identifier can have multiple proof entries in Uniqueproofs. We need to make sure all of them are removed. The list of digest belong to an identifier can be fetched from commits